app/features/send/cashu-send-quote-repository.ts

```diff
@@ -226,17 +238,22 @@ export class CashuSendQuoteRepository {
 
     const totalFees = actualLightningFee.add(quote.cashuFee);
 
+    const transaction = await this.transactionRepository.get(
+      quote.transactionId,
+    );
+
+    if (
+      !transaction ||
+      transaction.type !== 'CASHU_LIGHTNING' ||
+      transaction.direction !== 'SEND' ||
+      transaction.state !== 'PENDING'
+    ) {
+      throw new Error('Transaction not found.');
```

Comment:
I would add send quote id to the error. otherwise it would be very hard to debug this